plugins/woo-ai/src/product-description/product-description.scss

```diff
@@ -24,6 +24,9 @@
 	.components-snackbar__action:has( .woo-ai-feedback-snackbar-action) {
 		text-decoration: none;
 	}
+	div.woocommerce-layout__footer .woocommerce-transient-notices {
+		position: absolute !important;
```

Comment:
Hopefully this will be fixed by Ghidorah pretty quick 🤔. I wonder if this style would end up breaking whatever they were trying to fix; should we see how they respond before we commit something like this? If it's possible to target our snackbar notices specifically that would be an option as well, but I'm uncertain if we can do that.

Reply:
It's a little difficult to narrow this selector with the given markup, but we could add the body class `.post-type-product` to at least narrow it to the product editor.

Reply:
I'd love to not include this CSS at all, but I'm not seeing any PRs in the `wc-calypso-bridge` repo yet for a fix.
I've updated the selector to `body.post-type-product .woocommerce-transient-notices`, good idea. 👍